app/controllers/public_health_controller.rb

```diff
@@ -174,9 +174,10 @@ def sort(patients, order, direction)
     when 'dob'
       patients = patients.order('CASE WHEN date_of_birth IS NULL THEN 1 ELSE 0 END, date_of_birth ' + dir)
     when 'end_of_monitoring'
-      patients = patients.order('CASE
-      WHEN continuous_exposure = 1 THEN ' + (dir == 'asc' ? 'now()' : 'DATE("1970-01-01")') +
-      ' WHEN last_date_of_exposure IS NULL THEN patients.created_at ELSE last_date_of_exposure END ' + dir)
+      # `Continuous Exposure` values need a date associated with them so they always sort at the bottom. When sorting `desc`, using the epoch() time does the trick.
+      # But for `asc`, we want to use a time that logically comes after any dates in the system.
+      # System dates can be between now() and (30 days + the monitoring_period_days). The extra 1 (31 instead of 30) is to rule out any off-by-one errors.
```

Comment:
Is this comment still up to date?

Reply:
I could've sworn that I removed that comment. Must be losing my mind. Will remove now.